compose/src/main/java/com/maplibre/compose/ramani/MapCameraUpdater.kt

```diff
@@ -80,55 +82,76 @@ private class CameraTransitionListener(val map: MapLibreMap, val zoom: Double?,
   }
 }
 
-private class MapPropertiesNode(
-    val map: MapLibreMap,
-    var cameraPosition: MutableState<CameraPosition>
-) : MapNode {
+private class MapPropertiesNode(val map: MapLibreMap, var camera: MutableState<MapViewCamera>) :
+    MapNode {
   override fun onAttached() {
-    cameraUpdate(map, cameraPosition.value)
+    cameraUpdate(map, camera.value)
   }
 }
 
-private fun cameraUpdate(map: MapLibreMap, cameraPosition: CameraPosition) {
-  val cameraUpdate = CameraUpdateFactory.newCameraPosition(cameraPosition.toMapbox())
+private fun cameraUpdate(map: MapLibreMap, camera: MapViewCamera) {
+  val cameraUpdate = CameraUpdateFactory.newCameraPosition(camera.toCameraPosition())
+
+  // Handle values for all cases not in CameraPosition (pitchRange)
+  // TODO: Test how an invalid pitch value is handled when pitchRange is more restrictive.
```

Comment:
Revise comment to say this should never be valid because of the valid checks on MapViewCamera/CameraState.